ui/src/main/java/com/hyperwallet/android/ui/view/widget/DateWidget.java

```diff
@@ -50,11 +50,15 @@ public DateWidget(@NonNull HyperwalletField field, @NonNull WidgetEventListener
     public View getView() {
         if (mContainer == null) {
             mContainer = new RelativeLayout(mContext);
-            mTextInputLayout = new TextInputLayout(
-                    new ContextThemeWrapper(mContext, R.style.Widget_Hyperwallet_TextInputLayout));
+            mTextInputLayout = mField.isEditable() ? new TextInputLayout(new ContextThemeWrapper(mContext,
+                    R.style.Widget_Hyperwallet_TextInputLayout))
+                    : new TextInputLayout(new ContextThemeWrapper(mContext,
+                            R.style.Widget_Hyperwallet_TextInputLayout_Disabled));
```

Comment:
@peter-joseph  What do you think about extract themeWrapper?
             final ContextThemeWrapper themeWrapper = new ContextThemeWrapper(mContext, mField.isEditable() ?
                    R.style.Widget_Hyperwallet_TextInputLayout
                    : R.style.Widget_Hyperwallet_TextInputLayout_Disabled);
            mTextInputLayout = new TextInputLayout(themeWrapper); ?
Or we can extract it to AbstractWidget and remove this code from all widgets:     
protected ContextThemeWrapper getThemeWrapper(Context context, boolean isEditable) {
        return new ContextThemeWrapper(context, isEditable ?
                R.style.Widget_Hyperwallet_TextInputLayout
                : R.style.Widget_Hyperwallet_TextInputLayout_Disabled);
    }
and just call: mTextInputLayout = new TextInputLayout(getThemeWrapper(mContext, mField.isEditable()));

Reply:
hmm looks the same to me, unless we make a context wrapper utility